powsybl-parent/pom.xml

```diff
@@ -410,6 +410,27 @@
             </properties>
             <build>
                 <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>${maven.buildhelper.version}</version>
+                        <executions>
+                            <execution>
+                                <id>regex-property</id>
+                                <goals>
+                                    <goal>regex-property</goal>
+                                </goals>
+                                <configuration>
+                                    <!-- Change '\' by '/' in ${project.basedir} to allow lombok configuration on Windows -->
+                                    <name>escaped.project.basedir</name>
```

Comment:
nitpick but this is not escaping, more like a "unix conversion"

Reply:
This plugin execution was removed (see other comments).